### PR TITLE
Setting up MX records for my email

### PR DESCRIPTION
**Pull request by oliik2013**
for my email

### CODE REVIEW

File: domains/oliik2013.json [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
         },
     
         "record": {
-            "CNAME": "oliik2013.github.io"
+            "A": [
+              "185.199.108.153",
+              "185.199.109.153",
+              "185.199.110.153",
+              "185.199.111.153"
+            ]
```
Comment (CuteDog5695) [inline domains/oliik2013.json:15]:
```suggestion
            ],
```

File: domains/oliik2013.json [hunk 2]
```diff
@@ -1,13 +1,23 @@
...
+            "MX": [
+              "mx1.improvmx.com",
+              "mx2.improvmx.com"
+            ]
```
Comment (CuteDog5695) [inline domains/oliik2013.json:19]:
```suggestion
            ],
```

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/oliik2013.json (hunk 2 above)
Comment (oliik2013) [reply domains/oliik2013.json:19]: commited

**Comment by oliik2013** [comment]
now will be ok?

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Pull request merged**